cpp/example_code/s3/put_object_async.cpp

```diff
@@ -113,58 +111,63 @@ bool AwsDoc::S3::PutObjectAsync(const Aws::S3::S3Client& s3Client,
 }
 // snippet-end:[s3.cpp.put_object_async.code]
 
+/**
+ *
+ * main function
+ *
+ * Prerequisites: The bucket and the object to get the ACL information about.
```

Comment:
* Prerequisites: Create an S3 bucket that contains the object for which to get ACL information.